docs/getting-started.qmd

```diff
@@ -1,11 +1,11 @@
 ---
 title: "iBridges configuration"
 ---
-The iRODS GUI works with any iRODS instance. Here we show how to configure the application to work your specific iRODS instance.
+The iRODS GUI works with any iRODS instance. Here, we show how to configure the application to work your specific iRODS instance.
```

Comment:
"any iRODS instance" implies to me **any iRODS _version_**.  You may want to clarify that.